src/views/ScorePage/ScorePage.js

```diff
@@ -0,0 +1,86 @@
+import './ScorePage.css';
+import { scoreLocalStorage, saveHighScore } from '../../score/user-score';
+import Button from '../../components/Button';
+import '../../components/Button.css';
+
+export function renderScorePage() {
+  const highScores = JSON.parse(localStorage.getItem('highScores')) || [];
+  const recentUserScore = 12;
+  const questionsNumber = 12;
+
+  const container = document.createElement('div');
+  container.setAttribute('id', 'scorePage');
+  container.append(
+    renderGraphics(),
+    RenderScore(recentUserScore, questionsNumber),
+    renderNickForm(),
+    renderMenuButton(),
+    renderPlayAgainButton(),
```

Comment:
Keep the naming consistent. eg: 

RenderScore => renderScore
or 
Score (capital letter - as a 'component')

Reply:
and remember to stick with one naming convention